Highlight selected image

sarahlshowers opened the pull request:
- JS for event handler
- JS to toggle class
- CSS for styling border
- HTML add IDs to images

sarahlshowers committed:
Highlight selected image
- JS for event handler
- JS to toggle class
- CSS for styling border
- HTML add IDs to images

sarahlshowers committed:
resolve merge conflict

sarahlshowers commented:
@derrickshowers need this merged to build on JS

derrickshowers commented on app/wp-content/themes/kstg/library/js/vehicle.js:
```diff
@@ -41,7 +53,34 @@ var calloutValue = $('.specialFeature').text();
+      //event handler for image selection
+      $('img').on('click', setActiveImage)
+    }
     $('.next').on('click', function() {
```
What is this for? It's seems like it's doing the same thing as the handler on line 57.

derrickshowers commented on app/wp-content/themes/kstg/library/js/vehicle.js:
```diff
@@ -7,6 +7,18 @@ var calloutValue = $('.specialFeature').text();
+
+    //function for highlighting selected image in carousel
+    function setActiveImage(event) {
+      var currentImage = $('ul').find('.borderSelector').toggleClass('borderSelector');
```
This isn't quite right. What you're doing makes sense, but you wouldn't assign this to a variable. You could assign just `$('ul').find('.borderSelector')` to a variable (and then use that variable by doing something like `$var.toggleClass..`, if you wanted to, but in this situation there isn't really a need (you would want to do it if you're doing multiple things with that same element so that you don't have to search the DOM every time).

derrickshowers commented on app/wp-content/themes/kstg/library/js/vehicle.js:
```diff
@@ -7,6 +7,18 @@ var calloutValue = $('.specialFeature').text();
+    function setActiveImage(event) {
+      var currentImage = $('ul').find('.borderSelector').toggleClass('borderSelector');
+      console.log(currentImage);
+      var $getImage = (event.target.id);
```
It doesn't look like you're ever using this variable. 

FYI, the name is a little funny. I would name it something like `image` instead. If you were storing a `function` that gets an image, then the `getImage` name would make sense.

derrickshowers commented on app/wp-content/themes/kstg/library/js/vehicle.js:
```diff
@@ -41,7 +53,34 @@ var calloutValue = $('.specialFeature').text();
       }
     }
+    function setEventListeners() {
+      $('.next').on('click', function() {
```
Maybe not something that you need to do right away, but consider doing something like you did for the image click handler and move these into functions that get called above. Example:

`$('.next').on('click', advanceCarousel)`

You don't have to, but it would just keep your `setEventListeners` function clean so that wen someone looks at it, they can immediately get an idea as to what you're adding listeners to (the point of the function).

derrickshowers commented on app/wp-content/themes/kstg/page-vehicle.php:
```diff
@@ -32,22 +32,22 @@
         <ul>
           <?php if( !empty($image1) ): ?>
-            <li><img src="<?php echo $image1['url']; ?>" alt="<?php echo $image1['alt']; ?>" width="200px" height="100px" /></li>
+            <li><img id="img1" src="<?php echo $image1['url']; ?>" alt="<?php echo $image1['alt']; ?>" width="200px" height="100px" /></li>
```
Consider making this `id` something like 'img1-thumbnail' to differentiate it from the big images.

sarahlshowers committed:
Merge branch 'master' into 0804gallery

sarahlshowers committed:
Code Review Updates
Remove duplicate code
Delete unused vairbale name
Change id value to be more semantic

sarahlshowers committed:
rename variable

sarahlshowers committed:
remove unused variable

sarahlshowers committed:
resolve merge conflict
